### PR TITLE
Full Russian Translation [Updated]

### PR DESCRIPTION
**Pull request by aayojezuzchrezt**
Updated locales (synced and translated them after last update). (Not sure if I get "Good rankings", so left original next to translated, might fix it later)

### CODE REVIEW

File: website/public/locales/ru/side_menu.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-  "status": "Статус",
-  "status_dashboard": "Панель состояния системы",
+  "leaderboard": "Рейтинг",
   "user_leaderboard": "Таблица лидеров",
   "users": "Пользователи",
-  "users_dashboard": "Панель управления пользователями"
+  "users_dashboard": "Панель управления пользователями",
+  "status": "Статус",
+  "status_dashboard": "Панель состояния системы",
+  "trollboard": "Дрска позора"
```
Comment (Klotske) [inline website/public/locales/ru/side_menu.json:12]: Small typo :)
Should be "Доска"

**Comment by AbdBarho** [comment]
Blocked by #1327

**Review by Klotske — APPROVED** [review]
LGTM

**Comment by aayojezuzchrezt** [comment]
Done, I guess

**Review by fozziethebeat — CHANGES_REQUESTED** [review]
Will approve after the new merge conflicts are resolved

**Review by AbdBarho — APPROVED** [review]

**Review by fozziethebeat — APPROVED** [review]

**Pull request merged**